### ISSUE TITLE
Removing Unused CSS

### ISSUE THREAD

**Issue by bokster** [wvuweb/wvu-design-system#36]
Any plans to remove unused CSS so that the final CSS file is not bulky? Maybe something like [Postcss Uncss](https://github.com/uncss/postcss-uncss) or similar. Trying to think of ways to optimize output.

**Comment by adamglenn** [comment]
I'm not sure that will work since we are using a utility-based framework and you don't know which utilities are going to be applied on any given page. What we can recommend is that if developers know ahead of time which parts of Bootstrap they want to use, they can choose only those parts: https://getbootstrap.com/docs/4.3/getting-started/theming/ instead of linking to all of Bootstrap. I think the average site could knock off a big chunk of the file size. If you can do a demo of what you're talking about though I'm open to suggestions.

**Comment by bokster** [comment]
Ah, I didn't know you could choose utilities. I was coming from the "whole-kitchen-sink" point of view.

**Comment by adamjohnson** [comment]
Gonna close this.

tl;dr Go into your `styles.scss` file and remove any unused partials from being included in the compiled CSS.
